models/api.models.js

```diff
@@ -0,0 +1,7 @@
+const endpoints = require("../endpoints.json");
+
+function openEndpoints() {
+  return endpoints;
+}
+
```

Comment:
You wouldn't need a model for this as you can directly import the JSON file where needed.